tools/clang/unittests/SPIRV/CodeGenSPIRVTest.cpp

```diff
@@ -233,6 +233,7 @@ TEST_F(FileTest, FunctionCall) { runFileTest("fn.call.hlsl"); }
 
 // For early returns
 TEST_F(FileTest, EarlyReturn) { runFileTest("early-return.hlsl"); }
+TEST_F(FileTest, EarlyReturnFloat4) { runFileTest("early-return-float4.hlsl"); }
```

Comment:
I'd suggest to name these files as `cf.return.early{.float4|}.hlsl` so it will be more categorized. :)